linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/Emitter.scala

```diff
@@ -275,13 +285,10 @@ final class Emitter(config: Emitter.Config) {
              * causing JS static initializers to run. Those also must not observe
              * a non-initialized state of other static fields.
              */
-            topLevelExports ++
+            topLevelExports.iterator ++
```

Comment:
Should this go into the next commit?

Reply:
No, it's a boy scout change. `topLevelExports` is already a `List` here. It works at the right-hand-side of `Iterator.++`, but it was the only non-`Iterator` operand, so I made it explicit here. Does that make sense? I can also revert the change if you prefer.